python-sdk/example_dags/example_mysql_transform.py

```diff
@@ -0,0 +1,86 @@
+from datetime import datetime
+
+import pandas as pd
+from airflow import DAG
+
+from astro import sql as aql
```

Comment:
@phanikumv Do we need this?

Reply:
@utkarsharma2 yes  we are showing how to execute transformations in MySQL, you dont think its needed?

Reply:
@phanikumv Are we planning to add this in docs? or is there anything different from the transformation in other DB?

Reply:
nothing different @utkarsharma2 - will delete it